_posts/2023-04-18-searchable-snapshots.md

```diff
@@ -26,101 +26,34 @@ To enable searchable snapshots, reference the [instructions](https://opensearch.
 
 ## Using a snapshot for search
 
-OpenSearch cluster nodes retrieve data from a snapshot and then cache it to the node's local storage. You can configure the amount of storage available for caching snapshot data or use the system defaults. Any subsequent searches on the cached data perform better because they are served from the node's local storage, or cache, without the need to redownload the segment data from the snapshot.
+OpenSearch cluster nodes retrieve data from a snapshot and then cache it to the node's local storage. Any subsequent searches on the cached data perform better because they are served from the node's local storage, or cache, without the need to redownload the segment data from the snapshot.
 
-For a search request on data in an uncached snapshot, the data is downloaded from the snapshot's object store. If a node's cache is full, OpenSearch clears data segments from the cache using the Least Recently Used (LRU) caching strategy, removing the least recently used segments first.
+For a search request on data in an uncached snapshot, the data is downloaded from the snapshot's object store. If a node's cache is full, OpenSearch clears data segments from the cache using the Least Recently Used (LRU) caching strategy.
 
 ## Performance
```

Comment:
we need a "when to use Searchable Snapshot" section. I don't think we need a "when not to use" because its fairly obvious that its just the inverse of when to use. @nandi-github - please confirm. 
When to use:
"With less frequently accessed data which is not latency sensitive"

Reply:
Agree, we can add "When to use searchable Snapshot" section.  I currently could not find "when not to use". If it does not exist, then it is only matter of adding the when to use.